File: java/Throws.java
import java.io.*;

class Throws {
    public static void bacaFile() throws FileNotFoundException {
        throw new FileNotFoundException("File tidak ditemukan");
    }

    public static void main(String[] args) {
        try {
            bacaFile();
        } catch (IOException e) {
            System.out.println("Terjadi suatu error : " + e.getMessage());
        }
    }
}

File: cpp/Throws.cpp
#include <iostream>
#include <stdexcept>

// kita tau ini bakal ngethrow
void bacaFile() {
    throw std::runtime_error("File tidak ditemukan");
}

// jadi dihandling via try-catch
int main() {
    try {
        bacaFile();
    } catch (const std::exception& e) {
        std::cerr << "Terjadi suatu error: " << e.what() << std::endl;
    }

    return 0;
}